src/components/analytics/metrics-table.vue

```diff
@@ -133,6 +132,16 @@ export default {
       "num_offline_entity_submissions_reprocessed": "Number of reordered Offline Entity submissions",
       "max_entity_submission_delay": "Maximum Entity Submission processing delay",
       "avg_entity_submission_delay": "Average Entity Submission processing delay",
+      "uses_external_blob_store": "Server uses external blob store",
+      "num_offline_entity_submissions_force_processed": "Number of force-processed Offline Entity submissions",
+      "max_entity_branch_delay": "Maximum Offline Entity branch processing time",
+      "num_xml_only_form_defs": "Number of XML-only Form Defs with no associated XLSForm",
+      "num_blob_files": "Number of binary files",
+      "num_blob_files_on_s3": "Number of binary files currently stored in external Blob store",
+      "num_reset_failed_to_pending_count": "Number of times user has reset failed External Blob upload state",
```

Comment:
Might be OK to make this one a little shorter?

```suggestion
      "num_reset_failed_to_pending_count": "Number of times user has reset failed upload state",
```